catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_chm.jsonnet

```diff
@@ -0,0 +1,192 @@
+local id = 'projects/neon-prod-earthengine/assets/chm';
+local subdir = 'neon-prod-earthengine';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+local units = import 'units.libsonnet';
+
+{
+  // TODO(schwehr): Remove when the dataset is ready.
+  'gee:skip_indexing': true,
+
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'NEON Canopy Height Model (CHM)',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    Height of the top of canopy above bare earth (Canopy Height Model; CHM).
+    The CHM is derived from the LiDAR point cloud, and is generated by creating 
+    a continuous surface of canopy height estimates across the entire spatial 
+    domain of the LiDAR survey. The point cloud is separated into classes 
+    representing the ground and vegetation returns. The ground classified 
+    points are used to generate a height-normalized point cloud that
+    provides a relative estimate of vegetation elevation. A surface is then
+    generated using the height normalized vegetation points to produce the
+    CHM. Any canopy heights less than 2m are set to zero for data collected with
+    the Optech Gemini LiDAR sensors, and less than 2/3m for the Riegl and Optech
+    Galaxy sensors. Data are mosaicked over the AOP footprint onto a spatially 
+    uniform grid at 1 m spatial resolution. 
+
+    See [NEON Data Product DP3.30015.001](
+    https://data.neonscience.org/data-products/DP3.30015.001) for
+    more details.
+
+    Documentation: [NEON Ecosystem structure (DP3.30015.001) Quick StartGuide](
+    https://data.neonscience.org/api/v0/documents/quick-start-guides/NEON.QSG.DP3.30015.001v1?inline=true&fallback=html)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(license.reference),
+    // TODO(bhass-neon): Add doi link.
+    // {
+    //   rel: ee_const.rel.cite_as,
+    //   href: '',
+    //   type: ee_const.media_type.html,
+    // },
+  ],
+  keywords: [
+    # 'airborne',
+    # 'canopy',
+    # 'ecology',
+    'forest',
+    'lidar',
+    # 'neon',
+    'vegetation',
+  ],
+  providers: [
+    ee.producer_provider('NEON', 'https://www.neonscience.org/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-170, 16, -66, 73, '2013-01-01T00:00:00Z', null),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'AOP_VISIT_NUMBER',
+        description: 'Unique visit number to the NEON site',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'CITATION',
```

Comment:
What do you mean "it" - AOP_VISIT_NUMBER? This does change from asset to asset, this is the # of times we have flown over (or visited) a site. This is also currently in the asset name (eg. YEAR_SITE_VISIT), so it is a bit redundant here, but we thought it could potentially be a handy search parameter.

Reply:
Sorry, this was regarding the CITATION field. Also about DOI below.

Reply:
The CITATION and DOI will vary, depending on whether the data are Provisional or Released, and the Release Tag (eg. year of the Release).